### ISSUE TITLE
Example source code

### ISSUE THREAD

**Issue by borisoff** [222464/LTBL2#9]
Hi!

Really impressed with the original LTBL you adopted for SFML 2.5.1. Trying to use the LTBL2 with no luck. Can you post a complete source code for an example similar to the original LTBL one?

Thank you!

**Comment by eXpl0it3r** [comment]
I've uploaded the code to a repo of mine a very long time ago: https://github.com/eXpl0it3r/LTBL

**Comment by borisoff** [comment]
Yes, I checked that, and it works fine. I tried to use LTBL2 which has a different API. I can't figure out how to draw the shadows. Tried to draw LightSystem::getLightingTexture() texture with a sprite on top of my scene. But it just looks black for directional light, and has a small rect for point light.

**Comment by eXpl0it3r** [comment]
Ah, sorry. I misunderstood your question, I thought you wanted the LTBL source code, but you're actually looking for a LTBL 2 example 😅 

I see on the SFML forum there was a demo, sadly it didn't come with the source code 🙁

**Comment by 222464** [comment]
Hi, I can put aside some time to write an example, might take a bit though.

**Comment by borisoff** [comment]
Would be great!

**Comment by naftali100** [comment]
🙏

**Comment by pb-kh** [comment]
🙏
